code/modules/mob/living/carbon/human/npc/goblin.dm

```diff
@@ -287,10 +288,20 @@
 /datum/outfit/job/roguetown/npc/goblin/pre_equip(mob/living/carbon/human/H)
 	..()
 	H.STASTR = 8
+	var/chance_zjumper = 5
+	var/chance_treeclimber = 30
 	if(is_species(H, /datum/species/goblin/moon))
 		H.STASPD = 16
+		chance_zjumper = 20
+		chance_treeclimber = 70
 	else
 		H.STASPD = 14
+	if(prob(chance_zjumper))
+		ADD_TRAIT(H, TRAIT_ZJUMP, TRAIT_GENERIC)
```

Comment:
fwiw the zjumper stuff doesn't seem to work i don't think? i never got a chance to test and debug it because the main multiz map takes a fucking century to load